backend/apps/ifc_validation_bff/views_legacy.py

```diff
@@ -304,6 +304,13 @@ def upload(request):
         
         set_user_context(user)
 
+        referrer = request.headers["Referer"]
+        if referrer is not None:
+            if ("http://localhost" in referrer) or ("buildingsmart.org" in referrer):
```

Comment:
1. I had another look at the referrer policies that I vaguely remembered that could impact this. I don't think they will. In case of cross origin (which isn't even the in our deployed version) we only get the origin, which is still sufficient for this case.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Referrer-Policy#strict-origin-when-cross-origin_2

2. I do wonder though whether maybe we should only check for localhost in a development config? We have `PUBLIC_URL` in our .env file, can't we use that?